sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -379,9 +379,18 @@ def run_pipeline(self, pipeline, options):
     if apiclient._use_sdf_bounded_source(options):
       pipeline.replace_all(DataflowRunner._SDF_PTRANSFORM_OVERRIDES)
 
+    from apache_beam.portability.api import beam_runner_api_pb2
+    default_container_image = (
+        apiclient.get_default_container_image_for_current_sdk(
+            apiclient.get_job_type(options)))
```

Comment:
This value is only used for FnApi. Additionally, FnApi doesn't have different containers for streaming vs. batch. It's too bad we have to go through the (irrelevant for portability) job type to get this.

Reply:
Updated function to take a bool (use_fnapi) instead of job type.